contracts/DistributionCreator.sol

```diff
@@ -431,14 +404,21 @@ contract DistributionCreator is UUPSHelper, ReentrancyGuardUpgradeable {
         emit FeeRecipientUpdated(_feeRecipient);
     }
 
-    /// @notice Sets the message that needs to be signed by users before posting rewards
-    function setMessage(string memory _message) external onlyGovernor {
+    /// @notice Sets the message that needs to be accepted by users before posting rewards
+    function setMessage(string memory _message) external onlyGovernorOrGuardian {
         message = _message;
         bytes32 _messageHash = ECDSA.toEthSignedMessageHash(bytes(_message));
         messageHash = _messageHash;
         emit MessageUpdated(_messageHash);
     }
 
+    /// @notice Sets the defaultFees on deposit
+    function setFees(uint256 _defaultFees) external onlyGovernorOrGuardian {
```

Comment:
Better to couple with fee switch on the engine

Reply:
doesn't cost much to keep this fallback if engine fees do not work well -> not confident removing it if we haven't tested it well enough